### ISSUE TITLE
Problem with option 'remove_style_tags=True'

### ISSUE THREAD

**Issue by midezz** [Stranger6667/css-inline#110]
Hello!

There is a problem when we use css_inline with the option `remove_style_tags=True`. This problem appears when there are a few `<style>` tags in HTML.

HTML
```html
<html>
<head>
<style>
body {
    margin: 0 auto !important;
    padding: 0 !important;
    height: 100% !important;
    width: 100% !important;
    background: #f1f1f1;
}
a {
    text-decoration: none;
}
</style>
<style>
.test-class {
        color: #ffffff;
}
a {
        color: #17bebb;
}
</style>
</head>
<body>
<a class="test-class" href="https://example.com">Test</a>
</body>
</html>
```
Use css_inline with `remove_style_tags=True`
```python
inliner = css_inline.CSSInliner(remove_style_tags=True)
html_without_style = inliner(html)
```
Result
```html
<html>
<head>
<style>
.test-class {
        color: #ffffff;
}
a {
        color: #17bebb;
}
</style>
</head>
<body style="height: 100% !important;margin: 0 auto !important;padding: 0 !important;width: 100% !important;background: #f1f1f1;">
<a class="test-class" href="https://example.com" style="text-decoration: none;">Test</a>
</body>
</html>
```
The result is not correct. As you can see, not all styles apply to tag `<a>`, and not all tags `<style></style>` are removed.

Use css_inline without any options.
```python
html_with_style = css_inline.inline(html)
```
Result
```html
<html>
<head>
<style>
body {
    margin: 0 auto !important;
    padding: 0 !important;
    height: 100% !important;
    width: 100% !important;
    background: #f1f1f1;
}
a {
    text-decoration: none;
}
</style>
<style>
.test-class {
        color: #ffffff;
}
a {
        color: #17bebb;
}
</style>
</head>
<body style="margin: 0 auto !important;width: 100% !important;height: 100% !important;background: #f1f1f1;padding: 0 !important;">
<a class="test-class" href="https://example.com" style="color: #ffffff;text-decoration: none;">Test</a>
</body>
</html>
```
The result is correct.

**Comment by Stranger6667** [comment]
Hi! Thanks for reporting, I'll take a look

**Comment by Stranger6667** [comment]
The fix is released in `0.7.1`

**Comment by midezz** [comment]
Thank you very much!
